store/tikv/region_cache.go

```diff
@@ -329,6 +332,63 @@ func (c *RegionCache) GetRPCContext(bo *Backoffer, id RegionVerID, replicaRead k
 	}, nil
 }
 
+// GetTiFlashRPCContext returns RPCContext for a region must access flash store. If it returns nil, the region
+// must be out of date and already dropped from cache or not flash store found.
+func (c *RegionCache) GetTiFlashRPCContext(bo *Backoffer, id RegionVerID) (*RPCContext, error) {
+	ts := time.Now().Unix()
+
+	cachedRegion := c.getCachedRegionWithRLock(id)
+	if cachedRegion == nil {
+		return nil, nil
+	}
+	if !cachedRegion.checkRegionCacheTTL(ts) {
+		return nil, nil
+	}
+
+	regionStore := cachedRegion.getStore()
+
+	tikvCnt := 0
+	for i, store := range regionStore.stores {
```

Comment:
Do we have the situation that 1 region have multiple TiFlash? 

should we need to take care failure in one of TiFlash store and loadbalance between multiple TiFlash node.

for example, region1 has 3 TiFlash node: a, b, c.

we should give chance to use b and c; and send request to b and c when a is failured?